### ISSUE TITLE
Detect when Fn::Sub is incorrectly used in Output Description

### ISSUE THREAD

**Issue by ps-jay** [awslabs/cfn-python-lint#304]
**cfn-lint version:
- cfn-lint 0.4.2
- cfn-lint 0.5.2

**Description of issue.**
cfn-lint did not detect invalid use of Fn::Sub in the description field of an Output.

e.g.  this template snippet results in an error:
```
...
   SomeOutput:
    Description: !Sub "The thing that relates to ${SomeRef}"
     Value: !GetAtt 'SomeRef.Attrib'
     Export:
       Name: !Sub "${AWS::StackName}-SomeOutput"
```

Error:
> Template format error: Every Description member must be a string.

**Comment by ps-jay** [comment]
So speedy! .... wow..  thanks! : )
